config/locales/idv/en.yml

```
@@ -186,7 +186,9 @@ en:
         review: Re-enter your %{app_name} password to protect your data
     troubleshooting:
       headings:
-        are_you_near: 'Are you near Washington, D.C.?'
+        are_you_near: 'Are you located near Washington, D.C. or
+
+          Baltimore, MD?'
```

As an alternative to styling this with `white-space: pre`, I wonder if we could use HTML, such as [the `<wbr>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr)?

```suggestion
        are_you_near_html: 'Are you located near Washington, D.C. or<wbr>Baltimore, MD?'
```

Because the string is used in JavaScript (React), there's a bit of extra work to make the HTML show up. Specifically, there is a [helper function `formatHTML`](https://github.com/18F/identity-idp/blob/688a0d104f23296459a3316bb5282e691d24c360/app/javascript/packages/react-i18n/index.js#L10-L53) in the `@18f/identity-react-i18n` package which allows for HTML to be inserted. Unfortunately, I just noticed that (a) it's not very well-documented outside this function DocBlock, and (b) apparently it doesn't work for self-closing (void) tags like `<wbr />`. I can try to see if I can get that fixed up, since I was the one who had implemented the helper.

---

> I can try to see if I can get that fixed up, since I was the one who had implemented the helper.

Fix at #7034